docs/src/administration/configuration.md

```diff
@@ -88,6 +88,14 @@ The following includes ThreadPool related parameters, which can be passed in thr
 - a PKCS#8 private key file in PEM format. e.g. /home/tidb/client-key.pem.
 - default: null
 
+#### tikv.tls.reload_interval
+- The interval in seconds to poll the change of SSL context, if a change detected, the SSL context will be rebuilded.
```

Comment:
Maybe it is always watching the change of cert file rather than SSL context.

Reply:
The SSL context includes certificate files and private key files.